openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -2453,28 +2456,39 @@ def assign_partial_charges(
         oechem.OEThrow.SetOutputStream(errfs)
         oechem.OEThrow.Clear()
 
+        oe_charge_method = getattr(oequacpac, charge_method["oe_charge_method"])
+
         # The OpenFF toolkit has always supported a version of AM1BCC with no geometry optimization
         # or symmetry correction. So we include this keyword to provide a special configuration of quacpac
         # if requested.
         if partial_charge_method == "am1bccnosymspt":
-            optimize = False
-            symmetrize = False
-            quacpac_status = oequacpac.OEAssignCharges(
-                oemol, charge_method["oe_charge_method"](optimize, symmetrize)
-            )
-        else:
+            kwargs = {
```

Comment:
(not blocking) 75% of my time in this review was spent checking that this rearrangement of the shifty business logic of this method into a marginally less shifty form doesn't have some unintended effect. I know my slowness on reviews is a big blocker for getting stuff done, so I figured this was worth mentioning in terms of process improvement (since this seems kinda orthogonal to the goal of this PR, and the diff is already pretty sizable)